scripts/config.py

```diff
@@ -584,14 +585,14 @@ def __init__(self, crypto_config=None, **kw):
         super().__init__()
         self.crypto_config = CryptoConfigFile(crypto_config)
         self.settings.update({name: Setting(active, name, value, section, self.crypto_config)
-                                for (active, name, value, section)
-                                in self.crypto_config.parse_file()})
+                              for (active, name, value, section)
+                              in self.crypto_config.parse_file()})
 
     def set(self, name, value=None):
         if name in UNSUPPORTED_FEATURE:
-            raise ValueError('Feature is unsupported: \'{}\''.format(name))
+            raise ValueError(f'Feature is unsupported: \'{name}\'')
```

Comment:
We didn't use f-strings because they were introduced in Python 3.6 and for a long time, parts of our CI ran Python 3.5. We only moved to Python 3.6 as the minimum version on the CI [last month](https://github.com/Mbed-TLS/mbedtls-test/pull/167). So it's now ok to use f-strings.

Note that in 2.28 and 3.6, we only require “Python 3” for `config.py` (but “Python 3.6” in 2.28 for `generate_test_code.py`, and “Python 3.8” in 3.6 for that and other user-accessible scripts such as generating PSA driver wrappers). I think it would be ok to require Python 3.6 for `config.py` in 3.6, but we should be more conservative in 2.28.

Reply:
So, to clarify, for the sake of 2.28 we should revert back to old school `.format()`?

Reply:
From a code-style consistency perspective is it better to avoid f-strings for now and then raise a PR later on that replaces instances of `.format()` with f-strings in one go? Perhaps after we end support for 2.28?

Reply:
Cool, fine with me.